src/test/java/com/sap/oss/phosphor/fosstars/model/score/oss/ArtifactReleaseHistoryScoreTest.java

```diff
@@ -139,17 +156,17 @@ public void testTwoMonthOldVersion() {
   @Ignore
   public void testTwoMonthOldVersionStable() {
```

Comment:
Since you update this test, could you please have a look at #511 as well?

Reply:
For me no test is failing (and the GH actions seems happy too).
In #511 there is also no description what methods fails and what the failure is.
@sourabhsparkala , @artem-smotrakov what failure did you encounter?